common/djangoapps/course_modes/views.py

```diff
@@ -176,10 +176,7 @@ def get(self, request, course_id, error=None):
                 user=request.user,
                 course_key=course_key
             ),
-            "course_duration_limit_enabled": CourseDurationLimitConfig.enabled_for_enrollment(
-                user=request.user,
-                course_key=course_key
-            ),
+            "course_duration_limit_enabled": CourseDurationLimitConfig.enabled_for_enrollment(user, course),
```

Comment:
Should this be `request.user`?

```suggestion
            "course_duration_limit_enabled": CourseDurationLimitConfig.enabled_for_enrollment(request.user, course),
```

Reply:
Yes, good catch.